json-definitions/v3/tech-record/get/hgv/complete/index.json

```
@@ -1090,24 +1090,42 @@
             "techRecord_adrDetails_permittedDangerousGoods": {
               "type": "array",
               "contains": {
-                "type": "string",
-                "not": {
-                  "enum": [
-                    "Explosives (type 3)"
-                  ]
-                }
+                "const": "Explosives (type 3)"
               }
             }
           },
           "required": [
             "techRecord_adrDetails_permittedDangerousGoods"
           ]
         },
-        "then": {
-          "not": {
+        "else": {
```

Isn't this if else, basically the same thing? Can they be merged together?

---

In what way? Unsure what you want merged etc?

---

so like, adding it to be just one else? since you said body dec isn't mandatory 
```suggestion
"if": {
						"properties": {
							"techRecord_adrDetails_permittedDangerousGoods": {
								"type": "array",
								"contains": {
									"enum": [
										"FP <61 (FL)",
										"AT",
										"Class 5.1 Hydrogen Peroxide (OX)",
										"MEMU",
										"Carbon Disulphide",
										"Hydrogen",
										"Explosives (type 2)",
										"Explosives (type 3)"
									]
								}
							}
						},
						"required": [
							"techRecord_adrDetails_permittedDangerousGoods"
						]
					},
					"then": {
						"not": {
							"required": [
								"techRecord_adrDetails_bodyDeclaration_type"
							]
						}
					}
```

---

not required = not allowed, so this would mean you cannot send the value